src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -161,7 +161,7 @@ def test_WINNF_FT_S_WDB_1(self, config_filename):
 
     # Inject the PAL database URL into the SAS UUT.
     self._sas_admin.InjectDatabaseUrl({'type': 'PAL', 'url': pal_database.getBaseUrl()+
-                                       config['palDatabaseConfig']['fileUrl']})
+                                       config['palDatabaseConfig']['filePath']})
```

Comment:
I think you need to change the value of 'fileUrl', not this line. I understand 'filePath' to be the local path to the file, not the URL that the DB server will be expecting. So please revert this change and update 'fileUrl' to be 'pal_db.json'.

Reply:
yes, you are right.